### PR TITLE
Update README_ja.md

### PR DESCRIPTION
**Pull request by hiraku0424**

### CODE REVIEW

**Comment by hiraku0424** [comment]
@takayukioda review me

**Comment by takayukioda** [comment]
@takayukioda review me

File: README_ja.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+The API will have 4 options for managing challenges;
+ - GET, which gets a challenge based on its id
+ - DELETE, deletes a challenge based on its id
+ - POST, creates a new challenge and returns its id
+ - PUT, updates a challenge based on its id
+
+To protect against any unauthorized access, all endpoints will be protected with HMAC authorisation of which the secret (key) will be "challenge-database-api"
+
+## Specifications
+ - All API endpoints will be the following: "/api/challenge/[Id]", where [Id] is optional
```
Comment (takayukioda) [inline README_ja.md:19]: I think endpoint should be plural: "/api/challenge**s**/[Id]"
I ask @NckHmml to fix the code.

**Comment by takayukioda** [comment]
LGTM

**Comment by NckHmml** [comment]
fyi, the translation probably should also be merged into the js branch (NodeJS solution) 
@takayukioda I fixed the plural in both branches (master and js)

**Comment by takayukioda** [comment]
@NckHmml I see, thanks for the information!
@hiraku0424 Do not delete this branch after merge, and create new PR for js branch.

**Comment by takayukioda** [comment]
@hiraku0424 fix me

**Pull request closed without merging**